src/Storages/MergeTree/MergeTreeSettings.cpp

```diff
@@ -52,6 +52,10 @@ namespace ErrorCodes
     M(Bool, load_existing_rows_count_for_old_parts, false, "Whether to load existing_rows_count for existing parts. If false, existing_rows_count will be equal to rows_count for existing parts.", 0) \
     M(Bool, use_compact_variant_discriminators_serialization, true, "Use compact version of Variant discriminators serialization.", 0) \
     \
+    /** Merge selector settings. */ \
+    M(Bool, merge_selector_use_blurry_base, false, "Special mode for merge selector which makes it more aggressive when the table reaches the threshold for the number of parts.", 0) \
```

Comment:
Let's make it separate selector after  https://github.com/ClickHouse/ClickHouse/pull/70647